src/AuthenticatedAPIClient/authInterface.js

```diff
@@ -87,13 +84,14 @@ export default function applyAuthInterface(httpClient, authConfig) {
 
   httpClient.ensureAuthencationAndCookies = (route = '') => {
     // Validate auth-related cookies are in a consistent state.
-    const cookies = new Cookies();
-    const hasAccessToken = !!cookies.get(httpClient.accessTokenCookieName);
-    const hasUserInfo = !!cookies.get(httpClient.userInfoCookieName);
-    if (!hasAccessToken) {
+    const accessToken = httpClient.getDecodedAccessToken();
+    const tokenExpired = httpClient.isAccessTokenExpired(accessToken);
+    if (!accessToken || tokenExpired) {
```

Comment:
I think if !accessToken then tokenExpired will be true anyway, right?

Reply:
Yes, thanks!